README.md

```diff
@@ -7,3 +7,83 @@ In the directory `app/`, we have a simple Python application. We want to start o
 We will do this using the time-series database system [Prometheus](https://prometheus.io), which uses a "pull" method to extract data from running applications. This means that the applications need to "export" their data, so that Prometheus is able to "scrape" the metric data from them. This is typically done via an HTTP endpoint (`/metrics`, by convention).
 
 We will use the [Prometheus Python client library](https://github.com/prometheus/client_python) to track metrics in our code.
+
+## Content
+
+### Section 1: Exposing metrics
+
+For this section, you can use `make dev` to install depencies and run the dev server (you will need pipenv installed on your machine). (Docker will be used later)
```

Comment:
might be useful to add a prerequisites section above with `pipenv`

Reply:
Is the comment 

```
(Docker will be used later)
```
a note for the hosts or participants?

Reply:
I remember it as something we would say to the participants to make it clear we didn't use docker here but I would say its not so necessary and will remove it :)